src/sentry/migrations/0654_rename_priority_sort_to_trends.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 5.0.2 on 2024-02-16 04:58
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBarApprox
+
+
+def rename_priority_sort_to_trends(apps, schema_editor):
+    # We need to use apps.get_model to ensure we get the correct model for the
+    # migration. This is especially important in the case of a renamed model.
+    SavedSearch = apps.get_model("sentry", "SavedSearch")
+    for search in RangeQuerySetWrapperWithProgressBarApprox(SavedSearch.objects.all()):
+        if search.sort == "priority":
+            search.sort = "trends"
+            search.save()
```

Comment:
NIt: You can use `.save(update_fields=["sort"])`  here to just update the one field

Reply:
Skipping this because I don't want to wait for CI again, but I'll keep this in mind!